app/views/internal/broadcasts/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="row my-3">
+  <div class="col">
+    <div class="list-group-flush w-100">
+    <a class="list-group-item flex-column align-items-start">
+      <h3 class="mb-2">Title: <%= @broadcast.title %></h3>
+      <h3>Broadcast Type: <%= @broadcast.type_of %></h3>
+      <h3>Content:</h3>
+      <p>
+        <%= @broadcast.processed_html %>
+      </p>
+      <h3>Last Active On: <%= @broadcast.active_status_updated_at&.strftime("%b %d, %Y %H:%M UTC") %></h3>
+      <h3>Status:</h3>
+      <h3>
+      <span class="badge badge-<%= @broadcast.active? ? "success" : "warning" %>">
+        <%= @broadcast.active? ? "Active" : "Inactive" %>
+      </span>
+      <span class="sr-only">Broadcast Status</span>
+      </h3>
+    </a>
+    <hr>
+      <% if @broadcast.processed_html %>
+        <div>
+          </p><strong>Preview</strong></p>
+          <div class="broadcast-wrapper visible <%= banner_class(@broadcast) %>">
+          <div class="broadcast-data">
+            <%= sanitize @broadcast.processed_html, attributes: %w[href style src] %>
+          </div>
+          </div>
+        </p><em>Please note: announcement broadcasts will render directly below the nav bar once activated.</em></p>
+        <div>
+      <% end %>
+      <hr>
+       <div class="row justify-content-end">
+        <%= link_to "Edit Broadcast", edit_internal_broadcast_path, class: "btn btn-primary" %>
```

Comment:
I think it might be nice to just move the "Destroy Broadcast" button here, so that you can save yourself one click! Right now, you have to click into the form and _then_ delete the broadcast, which seems like more more step than really necessary 😝

Reply:
Good call! I've made this change @vaidehijoshi. 😄